### PR TITLE
chore: add default issue templates

### PR DESCRIPTION
**Pull request by trivikr**
*Issue #, if available:*
N/A

*Description of changes:*
These templates will help JS SDK team to create bugs or feature requests with smithy-typescript

Examples bugs created:
* https://github.com/awslabs/smithy-typescript/issues/73
* https://github.com/awslabs/smithy-typescript/issues/74
* https://github.com/awslabs/smithy-typescript/issues/75

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by trivikr** [comment]
Ping!

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```
Comment (kstich) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Please remove everything under each of the headings as it's just extra content that whoever opens an issue will need to remove themselves. Each section should be the heading followed by two empty lines:

```md
**Heading**


**Another heading**
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Additional context**
```
Comment (kstich) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Can you add a section asking for 'Smithy dependency versions'? This one is relevant, especially so while in preview.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```
Comment (kstich) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Please remove everything under each of the headings as it's just extra content that whoever opens an issue will need to remove themselves. Each section should be the heading followed by two empty lines:

```md
**Heading**


**Another heading**
```

**Comment by rstormsf** [comment]
Ping @trivikr. Please fix all the issues specified in the comments.

**Pull request closed without merging**